README.md

```diff
@@ -1,4 +1,16 @@
-sb-vision
-=========
+# sb-vision
 
 Vision subsystem for SourceBots, built on top of AprilTags and OpenCV.
+
+## Installation
+
+```bash
+$ pipenv install --dev
+```
+
+## Running Tests
+
+```bash
+$ pipenv run python setup.py test
```

Comment:
Seriously? How is this better than being able to `./setup.py $thing`?

Reply:
It's not equivalent to that, it activates the virtual environment and runs it in one command.

Reply:
I realise it's not technically equivalent, though it is functionally the same with more typing and less tab-completion.